### PR TITLE
Introduce distinct file service type

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/2398
## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal [hunk 1]
```diff
@@ -21,7 +21,7 @@ listener file:Listener localFolder = new ({
     recursive: false
 });
 
-service "filesystem" on localFolder {
+service file:Service "filesystem" on localFolder {
```
Comment (daneshk) [inline compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal:24]: AFAIK we don't need this with the latest lang fix

Comment (daneshk) [reply compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal:24]: Can you with the latest lang version

Comment (shafreenAnfar) [reply compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal:24]: It should be `ballerinaLangVersion=2.0.0-beta.4-20211117-131600-4133d87c`

Comment (kalaiyarasiganeshalingam) [reply compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal]: Updated

**Review by daneshk — APPROVED** [review]

**Pull request merged**